portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,35 @@
+body {
+  /* background-image: url("/images/background.jpg");
+  background-size: cover; */
```

Comment:
Delete the commented out code if it is no longer necessary. It is better to clean up before submitting the code. That way you don't leave behind things that needs to be cleaned up later, which most people will forget to do.

Reply:
Thanks for the reminder